package.json

```diff
@@ -35,7 +35,9 @@
     "babel-preset-stage-0": "^6.3.13",
     "eslint": "^4.2.0",
     "eslint-plugin-react": "^7.0.0",
-    "webpack": "3.1.0",
+    "node-sass": "^4.5.3",
```

Comment:
Missing style-loader, css-loader dependencies. And please revert webpack version. This is the source of the issue.